### ISSUE TITLE
dreambooth training not working

### ISSUE THREAD

**Issue by Aresisok** [bmaltais/kohya_ss#906]
when i train i get this:
 "No [] were found in train_data_dir can't train..."
i have images in my new folder 42, they are named (1).png, (2).png, ect
i try renaming images but same error
settings
{
  "pretrained_model_name_or_path": "G:/Delta/stable-diffusion-webui/models/Stable-diffusion/2Mixsjt.ckpt",
  "v2": true,
  "v_parameterization": false,
  "logging_dir": "",
  "train_data_dir": "C:/Users/Admin/Downloads/New folder (42)",
  "reg_data_dir": "",
  "output_dir": "G:/Delta/stable-diffusion-webui/models/Stable-diffusion",
  "max_resolution": "768,768",
  "learning_rate": 1e-05,
  "lr_scheduler": "cosine",
  "lr_warmup": "10",
  "train_batch_size": 1,
  "epoch": 100,
  "save_every_n_epochs": 1,
  "mixed_precision": "fp16",
  "save_precision": "fp16",
  "seed": "",
  "num_cpu_threads_per_process": 2,
  "cache_latents": true,
  "cache_latents_to_disk": false,
  "caption_extension": "",
  "enable_bucket": true,
  "gradient_checkpointing": false,
  "full_fp16": false,
  "no_token_padding": false,
  "stop_text_encoder_training": 0,
  "xformers": true,
  "save_model_as": "safetensors",
  "shuffle_caption": true,
  "save_state": false,
  "resume": "",
  "prior_loss_weight": 1.0,
  "color_aug": false,
  "flip_aug": false,
  "clip_skip": "1",
  "vae": "",
  "output_name": "luuuubi",
  "max_token_length": "75",
  "max_train_epochs": "",
  "max_data_loader_n_workers": "0",
  "mem_eff_attn": false,
  "gradient_accumulation_steps": 1.0,
  "model_list": "custom",
  "keep_tokens": "0",
  "persistent_data_loader_workers": false,
  "bucket_no_upscale": true,
  "random_crop": false,
  "bucket_reso_steps": 64.0,
  "caption_dropout_every_n_epochs": 0.0,
  "caption_dropout_rate": 0,
  "optimizer": "AdamW8bit",
  "optimizer_args": "",
  "noise_offset_type": "Original",
  "noise_offset": 0.02,
  "adaptive_noise_scale": 0,
  "multires_noise_iterations": 0,
  "multires_noise_discount": 0,
  "sample_every_n_steps": 0,
  "sample_every_n_epochs": 0,
  "sample_sampler": "euler_a",
  "sample_prompts": "",
  "additional_parameters": "",
  "vae_batch_size": 0,
  "min_snr_gamma": 0,
  "weighted_captions": false,
  "save_every_n_steps": 0,
  "save_last_n_steps": 0,
  "save_last_n_steps_state": 0,
  "use_wandb": false,
  "wandb_api_key": ""
}

**Comment by E2GO** [comment]
Same

**Comment by bmaltais** [comment]
YEah... your folder structure is wrong. It need to be like:

`C:/Users/Admin/Downloads/New folder (42)/30_my dataset`

And the images go in the `30_my dataset folder`.

30 mean 30 repeats of all images on the folder.

**Comment by YOlegY** [comment]
Same issue.And making similar folder structure didn't helped (C:\Users\impos\Downloads\New Folder (1)\1_image of Cat Cat*) 
Inside folder two test files -  (1).png (1).txt

Png is 512*512
Other settings by default , except Epoch (100), Optimizer(Lion8Bit) and Noise Ofccet(0,1)
*Folder made by "Prepare training data" GUI , and no , changing to "30_my dataset folder" doesn't helps.

**Comment by YOlegY** [comment]
Helped clean reinstall.After that got similar error but with tip before it , it suggested to go in docs folder inside kohya_ss and read "image_folder_structure.md"(Notepad++ works with this file extension).
After that my image folder structure looks like this - C:\images\1_cat
And in Dreambooth GUI this needs to be selected like C:\images
